Dockerfile.rocm.ubi

```diff
@@ -178,7 +197,13 @@ RUN --mount=type=bind,from=build_amdsmi,src=/install,target=/install/amdsmi/ \
 RUN umask 002 && \
     useradd --uid 2000 --gid 0 vllm && \
     mkdir -p /licenses && \
-    chmod g+rwx $HOME /usr/src /workspace
+    chmod g+rwx $HOME /usr/src /workspace && \
```

Comment:
Why is this required?

Reply:
Fixes world writable directory permissions in the next line.

Reply:
OK, I think the permissions are right. Marking this resolved.